ert_shared/ensemble_evaluator/entity/snapshot.py

```diff
@@ -190,6 +190,13 @@ def to_dict(self):
     def get_status(self):
         return self._data["status"]
 
+    def is_finished(self):
+        for real in self._data["reals"].values():
+            for stage in real["stages"].values():
+                if stage["status"] != "Finished":
```

Comment:
As the stages are deemed finished by the `JobQueueManagerAdaptor`, this too can be true before the nfs adaptors have finished.